src/doc/book/loops.md

```diff
@@ -80,13 +80,16 @@ for var in expression {
 }
 ```
 
-The expression is an [iterator][iterator]. The iterator gives back a series of
-elements. Each element is one iteration of the loop. That value is then bound
-to the name `var`, which is valid for the loop body. Once the body is over, the
-next value is fetched from the iterator, and we loop another time. When there
-are no more values, the `for` loop is over.
+The expression is an item can can be converted into an
+[iterator][iterator] using [`IntoIterator`][intoiterator]. The
+iterator gives back a series of elements. Each element is one
+iteration of the loop. That value is then bound to the name `var`,
+which is valid for the loop body. Once the body is over, the next
+value is fetched from the iterator, and we loop another time. When
+there are no more values, the `for` loop is over.
 
 [iterator]: iterators.html
+[intoiterator]: ../std/iter/trait.IntoIterator.html
```

Comment:
you can make this 

    [`IntoIterator`]

and then drop the second set of []. actually, the first set can go too, since the text is the same. Mind doing that and reflowing to 80 cols? r=me after